idea-plugin/build.gradle.kts

```diff
@@ -20,17 +20,34 @@ dependencies {
 
     compileOnly(compose.desktop.currentOs) {
         exclude(group = "org.jetbrains.compose.material")
+        exclude(group = "org.jetbrains.kotlinx")
+    }
+    implementation(compose.desktop.common) {
+        exclude(group = "org.jetbrains.kotlinx")
+    }
+    implementation(compose.desktop.linux_arm64) {
+        exclude(group = "org.jetbrains.kotlinx")
+    }
+    implementation(compose.desktop.linux_x64) {
+        exclude(group = "org.jetbrains.kotlinx")
+    }
+    implementation(compose.desktop.macos_arm64) {
+        exclude(group = "org.jetbrains.kotlinx")
+    }
+    implementation(compose.desktop.macos_x64) {
+        exclude(group = "org.jetbrains.kotlinx")
+    }
+    implementation(compose.desktop.windows_x64) {
+        exclude(group = "org.jetbrains.kotlinx")
+    }
+    implementation(compose.material3) {
+        exclude(group = "org.jetbrains.kotlinx")
```

Comment:
Excluding `org.jetbrains.kotlinx` could be extracted to `configurations.configureEach` or something.

Reply:
tried, it's failing

![image](https://github.com/user-attachments/assets/0444b8e9-afac-4a27-87de-8e8b89465f63)

in this case I just excluded where its necessary